src/core/load-fragment.c

```diff
@@ -4122,6 +4122,39 @@ int config_parse_managed_oom_mem_pressure_limit(
         return 0;
 }
 
+int config_parse_managed_oom_mem_pressure_duration_sec(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        usec_t *duration = data;
+        usec_t usec;
+        UnitType t;
+        int r;
+
+        t = unit_name_to_type(unit);
+        assert(t != _UNIT_TYPE_INVALID);
+
```

Comment:
so we generally allow assigning an empty string to settings to reset them to defaults (this is useful in .d/ drop-ins, so that they can reset values the main unit file set to some value back to the defaults). Hence, could you add this here:

```c
if (isempty(rvalue)) {
        *duration = UINT64_MAX
        return 0;
}
```

(assuming UINT64_MAX is the new niche value for "unset"